### PR TITLE
Update InnerSource SIG README and Contributing.md header content

### LINKED ISSUE #4: InnerSource SIG to update current project README
### Description of Problem:
The current InnerSource README currently includes the default text of the FINOS Project Blueprint ...
- https://github.com/finos/InnerSource/blob/master/README.md

### Potential Solutions:
InnerSource Leadership team to contribute an updated README using the Regulation Innovation SIG as great example of how to describe the SIG, get people involved in meetings and contributing materials to the SIG repo ...
- https://github.com/finos/open-regtech-sig

### References
This requirement is also referenced in the FINOS Community contribution [#84](https://github.com/finos/community/issues/84)
> Update SIG README

### PR DESCRIPTION
**Pull request by mcleo-d**
## Description

This pull request closes #4 by updating the InnerSource SIG `README.md` and `Contributing.md` header content ready for the InnerSource SIG announcement email to FINOS.

### Included Commits 
- update InnerSource header 43a678c
- remove FINOS logo 2b4831c
- update README content 28c1cdc
- update contribute.md header 856c7ba
- update header with FINOS eb6342d
- update readme content 284a18f
- update readme content 46b3829
- update readme content d00d7a1

### Contributing.md Markdown Changes
from ...
`# {project name} Contribution and Governance Policies`
to
`# InnerSource SIG Contribution and Governance Policies`

### README Content Preview

# FINOS InnerSource Special Interest Group

InnerSource can help break down silos, encourage internal collaboration and innovation, accelerate new engineer on-boarding, and identify opportunities to contribute software back to the open source world.

## Introducting the InnerSource SIG

The FINOS InnerSource [Special Interest Group](https://github.com/finos/community/tree/master/governance#special-interest-groups) is a community of people implementing, or interested in implementing, InnerSource within their financial services organization. 

The InnerSource SIG is of particular importance to financial services organisations who wish to accelerate their InnerSource practices, share best practices, patterns and anti-patterns and potentially related code (e.g. for InnerSource tooling) in a secure environment, with the aim of contributing proven methods to FINOS and the wider InnerSource Commons community. 

Discussions have an emphasis on challenges and concerns that are particular to FS organizations (e.g. relating to compliance or regulatory constraints).

The InnerSource SIG target audience are individuals responsible for InnerSource implementation within financial services organizations, those in Open Source Program Offices interested in open source culture, behaviour and skills, or tech leaders wishing to increase collaboration and remove / deal with excessive ownership issues that can stall innovation.  

## InnerSource SIG Leadership

Name | Firm | Role
:--- | :--- | :---
Daniela Zheleva | Deutsche Bank | SIG Lead
Roderick Randolph | Capital One | SIG Co-Lead
Arthur Maltson | Capital One | SIG Co-Lead
Clare Dillon | InnerSource Commons | SIG Secretary
Aaron Searle | Morgan Stanley | SIG Leadership  Committee
Anthony Vacca | RBC | SIG Leadership  Committee
Danese Cooper | InnerSource Commons | SIG Leadership Committee

## Mailing List
All SIG related communications are conducted through the innersource@finos.org mailing list. Email innersource@finos.org with questions or suggestions for collaboration use cases. Join the mailing list and stay up to date by sending a note to innersource+subscribe@finos.org.

## Contributing
1. Fork it (https://github.com/finos/innersource/fork)
2. Create your feature branch (git checkout -b feature/fooBar)
3. Read our [contribution guidelines](https://github.com/finos/InnerSource/blob/master/CONTRIBUTING.md) and [Community Code of Conduct](https://www.finos.org/code-of-conduct)
4. Commit your changes (git commit -am 'Add some fooBar')
5. Push to the branch (git push origin feature/fooBar)
6. Create a new Pull Request

_NOTE:_ Commits and pull requests to FINOS repositories will only be accepted from those contributors with an active, executed Individual Contributor License Agreement (ICLA) with FINOS OR who are covered under an existing and active Corporate Contribution License Agreement (CCLA) executed with FINOS. Commits from individuals not covered under an ICLA or CCLA will be flagged and blocked by the FINOS Clabot tool. Please note that some CCLAs require individuals/employees to be explicitly named on the CCLA.

Need an ICLA? Unsure if you are covered under an existing CCLA? Email help@finos.org

## License

Copyright 2021 Fintech Open Source Foundation

Distributed under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).

SPDX-License-Identifier: [Apache-2.0](https://spdx.org/licenses/Apache-2.0)

### CODE REVIEW

**Review by daniela-g-zheleva-db — APPROVED** [review]
Looks great James 👍

**Comment by claredillon** [comment]
Looks great James- thanks! :)

Could I suggest also adding the following sentence at the bottom: "Where appropriate, the InnerSource SIG will also work with InnerSource Commons (www.innersourcecommons.org) to share relevant outputs upstream to the broader InnerSource community." 

It will help encourage participation from the ISC community too!

**Comment by mcleo-d** [comment]
> Looks great James- thanks! :)
> 
> Could I suggest also adding the following sentence at the bottom: "Where appropriate, the InnerSource SIG will also work with InnerSource Commons ([www.innersourcecommons.org](http://www.innersourcecommons.org)) to share relevant outputs upstream to the broader InnerSource community."
> 
> It will help encourage participation from the ISC community too!

Thanks @claredillon - I have committed your feedback as 8e0aa0c 👍

**Comment by mcleo-d** [comment]
This PR was merged based on @daniela-g-zheleva-db's approval and @claredillon's feedback being fulfilled here https://github.com/finos/InnerSource/pull/7#issuecomment-785207378

File: README.md [hunk 1]
```diff
@@ -1,23 +1,49 @@
-<img align="right" width="40%" src="https://www.finos.org/hubfs/FINOS/finos-logo/FINOS_Icon_Wordmark_Name_RGB_horizontal.png">
+# FINOS InnerSource Special Interest Group
 
-# FINOS Software Project Blueprint
+InnerSource can help break down silos, encourage internal collaboration and innovation, accelerate new engineer on-boarding, and identify opportunities to contribute software back to the open source world.
 
-Project blueprint is a GitHub repository template for all [Fintech Open Source Foundation (FINOS)](https://www.finos.org/) hosted GitHub repositories, contributed and maintained by FINOS as part of the [Open Developer Platform (ODP)](https://odp.finos.org) initiative.
+## Introducting the InnerSource SIG
```
Comment (amaltson) [inline README.md:5]: Should this be `Introducing`?

**Review by amaltson — COMMENTED** [review]
Just the one comment, otherwise this looks great! Thanks!

File: README.md (hunk 1 above)
Comment (mcleo-d) [reply README.md:5]: Thank you for your keen eyes @amaltson 👀 

I have raised PR #8 if you can review and merge ...

James.

**Pull request merged**